docs/build-with-powerloom/devnet/troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 3
+---
+
+# Troubleshooting
+
+
```

Comment:
All these points are present in snapshotter node troubleshooting I think. We should just link to those and add devnet specific things here like issue with minting, burner wallet, how to check your submissions, etc.